app/grants/views.py

```diff
@@ -1788,13 +1793,14 @@ def grant_new(request):
         harmony_payout_address = request.POST.get('harmony_payout_address', None)
         binance_payout_address = request.POST.get('binance_payout_address', None)
         rsk_payout_address = request.POST.get('rsk_payout_address', None)
+        algorand_payout_address = request.POST.get('algorand_payout_address', None)
 
         if (
             not eth_payout_address and not zcash_payout_address and
             not celo_payout_address and not zil_payout_address and
             not polkadot_payout_address and not kusama_payout_address and
             not harmony_payout_address and not binance_payout_address and
-            not rsk_payout_address
+            not rsk_payout_address and not algorand_payout_address
```

Comment:
It looks like we have a lot of places here where we have to check for every chain in these conditionals. Rather than repeating these over and over, could we make a set and then check the set? For example with these payout addresses, rather than doing this long list of `not`s would it be cleaner to add these to a set and then check if the set is empty?
```
all_payout_address_types = {eth_payout_address, zcash_payout_address, celo_payout_address, zil_payout_address, polkadot_payout_address, kusama_payout_address, harmony_payout_address, binance_payout_address, rsk_payout_address, algorand_payout_address}

if not all_payout_address_types:
    response['message'] = 'error: payout_address is a mandatory parameter'
```

Reply:
as synced up offline -> will do this in a follow up PR

Reply:
My bad. Should be `if not any(all_payout_address_types):`